### ISSUE TITLE
[Enhancement]: Virtual Joint for Octomap

### ISSUE THREAD

**Issue by AntoRag** [Interbotix/interbotix_ros_rovers#23]
### Describe the solution you'd like

Hi, I'm currently using the Locobot WX250s for my master's thesis. I was wondering if it was possible to modify the description of the robot and add a virtual joint between base footprint and a fixed frame (for example "map"). I had this necessities because i was trying to build the Octomap incrementally and in this issue suggested this solution [https://github.com/ros-planning/moveit/issues/1188]

### Platform

Locobot wx250s, Ubuntu 20.04, ROS Noetic

**Comment by LSinterbotix** [comment]
Yes, it is possible. Feel free to modify the [locobot.urdf.xacro](https://github.com/Interbotix/interbotix_ros_rovers/blob/main/interbotix_ros_xslocobots/interbotix_xslocobot_descriptions/urdf/locobot.urdf.xacro) in your local workspace.
